packages/firebase_storage/pubspec.yaml

```diff
@@ -3,7 +3,7 @@ description: Flutter plugin for Firebase Cloud Storage, a powerful, simple, and
   cost-effective object storage service for Android and iOS.
 author: Flutter Team <flutter-dev@googlegroups.com>
 homepage: https://github.com/flutter/plugins/tree/master/packages/firebase_storage
-version: 0.2.6
+version: 0.2.7
```

Comment:
Should be `0.3.0`, if we remove the `file` field of `StorageUploadTask`. I think we should; it is not entirely clear to me what clients would want that field for anyway. Add `**Breaking change**` in change log, if we go with that option.